### PR TITLE
Merging Dashboard into main

### PR DESCRIPTION
**Pull request by JPCatze**

### CODE REVIEW

File: lib/src/features/dashboard/models/modify_event.dart
Comment (INZDY) [inline lib/src/features/dashboard/models/modify_event.dart:1]: Change the file name, duplicate name with planner feature.
If the 2 files are identical, delete this one and use the one from planner.

**Review by INZDY — COMMENTED** [review]
I didn't check much in dashboard.
After finishing this, I'll need to pull and test on my machine for real usage.

File: lib/src/features/dashboard/models/modify_event.dart
Comment (JPCatze) [reply lib/src/features/dashboard/models/modify_event.dart:1]: ok, file name changed

**Review by INZDY — APPROVED** [review]
Ready for merging.

**Pull request merged**